lib/Doctrine/CouchDB/Utils/BulkUpdater.php

```diff
@@ -83,6 +84,22 @@ public function execute()
         return $this->httpClient->request('POST', $this->getPath(), json_encode($this->data), false, $this->requestHeaders);
     }
 
+    public function executeByLimit($limit = 100)
+    {
+        // Do multiple POST requests if the number of docs is higher than $limit.
+        $result = [];
+        foreach (array_chunk($this->data['docs'], $limit) as $data) {
+            $full_data = $this->data;
+            $full_data['docs'] = $data;
+            $response = $this->httpClient->request('POST', $this->getPath(), json_encode($full_data), false, $this->requestHeaders);
+            if ($response->status != 201) {
+                throw HTTPException::fromResponse($response, $this->getPath());
+            }
+            $result += $response->body;
```

Comment:
@damiankloip I don't understand how this ca yield more results than the regular bulk request. The difference between them is that the regular one will put all docs in one request and the new one will do multiple POST requests if the number of docs in the bulk is greater than 100 (the limit).